docs/quick-start/lua-howto.md

```diff
@@ -0,0 +1,95 @@
+---
+template: main.html
+---
+
+![Setup-Banner](https://github.com/ExpressLRS/ExpressLRS-Hardware/raw/master/img/quick-start.png)
+
+Now that you have flashed your TX it is time to learn how to use the ELRS lua script!
+
+*If you flashed via WiFi you need a different version of the lua than you already installed.*
+
+## Installing the Lua Script
+
+Download the [ELRSv2 Lua Script](https://github.com/ExpressLRS/ExpressLRS/tree/master/src/lua) for your specific radio type onto the radio's SD Card in the `Scripts/Tools` folder and Long Press the "SYS" button (for T16 or similar Radios) or the "Menu" button (for Taranis X9D or similar Radios) to access the Tools Menu where you can find ELRS script ready to run with only one click.
+
+Which version?
+
+* if your handset has the black and white display and your handset is NOT an X9D or X9D+, use: **black_and_white_display_128x64**
+
+* if your handset is the X9D or X9D+, use: **x9d_212x64**
+
+* if your handset has the colored screen display, use: **color_display_480x272**
+
+Here's how it looks in the Tools Menu (X9D+ and T16 Shown):
+
+<img src="../../assets/images/lua1.jpg" width ="40%">
+<img src="../../assets/images/lua2.jpg" width ="40%">
+
+## Check your TX is Connected
+
+Enter the Lua Script by short pressing it in the Tools menu. The next section is to check if your TX and Handset are communicating.
+
+If no parameter is shown for a long time in the Lua Script, that would means the Protocol set for the External Module is incorrect or that the module is not properly connected to the pins of the jr bay of the radio. The latter could mean that the module's PCB has gotten loose, common with the first batches of the ES24TX modules from Happymodel.
+
+If a parameter does show up, but the Bad/Good section on the right of the header from the lua script is showing 0/142, etc., have a look at your model settings and make sure the internal RF module is set to off.
+
+If the Bad/Good persists to be something different than `0/your packet rate` that means `CRSFshot` is not working - verify that you properly followed the [Radio Preparation](./../tx-prep) Guide.
```

Comment:
maybe it's better to flip the paragraph, to explain what is it first, and also rather than saying that the numbers showed "0/142,etc.. " it's better to say "if the number is unstable and keep changing"